src/components/SingleArticle.jsx

```diff
@@ -16,6 +18,10 @@ export default function SingleArticle() {
         getArticleById(article_id)
             .then((articleData) => {
                 setArticle(articleData)
+                return getCommentsByArticleId(article_id)
+            })
+            .then((commentsData) => {
+                setComments(commentsData)
                 setIsLoading(false)
             })
```

Comment:
The useEffect here is not refreshing when the comments change, so it posts to the database but you have to refresh the entire page to have the new comments be displayed, could you change something so that the comments component re-renders after a post ?